citar-org-roam.el

```diff
@@ -186,9 +186,15 @@ space."
   ;; adapted from https://jethrokuan.github.io/org-roam-guide/#orgc48eb0d
   (let ((title (citar-format--entry
                 citar-org-roam-note-title-template entry))
-        (key citar-org-roam-capture-template-key))
+        (key citar-org-roam-capture-template-key)
+        (author (citar-format--entry
+                 "${author}" entry))
```

Comment:
```suggestion
                 "${author editor}" entry))
```

So two minor things I'm not sure of, so leave you to think about.

1. the suggestion above
2. whether to use one of the new template modifiers to transform here?

```elisp
ELISP> (citar-format--entry "${author editor:%etal}" (citar-get-entry "francis2012"))
"Francis, Giles-Corti & Wood et al."
```